### ISSUE TITLE
to do list

### ISSUE THREAD

**Issue by BrunoLiegiBastonLiegi** [qiboteam/rtqem#39]
We report here the things we still need to do:

### code
- add training loss/ gradient saving in the training loop
- write a script to plot and compare training loss and gradient variance with and without mitigation

As we verified that the model is capable to learn the parameters for the fit even without mitigation, it's important to justify the use of the mitigation, for example, by demonstrating that the convergence is faster and or more stable.

### experiments
- we should decide the configuration for the model (`n_layers`, `n_shots`, ecc...) and the training (`learning_rate`, `n_data`, `n_epochs`, ecc...) for each one of the examples and keep it fixed. Then run the experiments on the hardware and collect the results.
- deploy test: train a model on one chip and then deploy it to another one, demonstrate that even though the non mitigated model works, it's not general as when the noise changes it is no longer usable. On the contrary, if we train a model with mitigation it should be noise independent, and, therefore, it allows to deploy the learnt model on any other quantum chip.

### paper 
- write the explanation of the CDR method
- start to write the experiments section by describing which functions we consider and what kind of experiments we run
- start drafting the introduction and or related work section with all the references

@AlejandroSopena @MatteoRobbiati did I forget anything?
